lib/usertasks/urls.go

```diff
@@ -172,3 +173,71 @@ func EC2InstancesWithURLs(ut *usertasksv1.UserTask) *UserTaskDiscoverEC2WithURLs
 		Instances:   instancesWithURLs,
 	}
 }
+
+// UserTaskDiscoverRDSWithURLs contains the databases that failed to auto-enroll into the cluster.
+type UserTaskDiscoverRDSWithURLs struct {
+	*usertasksv1.DiscoverRDS
+	// Databases maps a database resource id to the result of enrolling that database into teleport.
+	// For RDS Aurora Clusters, this is the DBClusterIdentifier.
+	// For other RDS databases, this is the DBInstanceIdentifier.
+	Databases map[string]*DiscoverRDSDatabaseWithURLs `json:"databases,omitempty"`
+}
+
+// DiscoverRDSDatabaseWithURLs contains the result of enrolling an AWS RDS Database.
+type DiscoverRDSDatabaseWithURLs struct {
+	*usertasksv1.DiscoverRDSDatabase
+
+	// ResourceURL is the Amazon Web Console URL to access this RDS Database.
+	// Always present.
+	// Format for instances: https://console.aws.amazon.com/rds/home?region=<region>#database:id=<name>;is-cluster=false
+	// Format for clusters:  https://console.aws.amazon.com/rds/home?region=<region>#database:id=<name>;is-cluster=true
+	ResourceURL string `json:"resourceUrl,omitempty"`
+
+	// ConfigurationURL is the Amazon Web Console URL that shows the configuration of the database.
+	// Format https://console.aws.amazon.com/rds/home?region=<region>#database:id=<name>;is-cluster=<is-cluster>;tab=configuration
+	ConfigurationURL string `json:"configurationUrl,omitempty"`
+}
+
+func withRDSDatabaseIssueURL(metadata *usertasksv1.UserTask, database *usertasksv1.DiscoverRDSDatabase) *DiscoverRDSDatabaseWithURLs {
+	ret := &DiscoverRDSDatabaseWithURLs{
+		DiscoverRDSDatabase: database,
+	}
+
+	fragment := fmt.Sprintf("database:id=%s;is-cluster=%t", database.GetName(), database.GetIsCluster())
+	databaseBaseURL := url.URL{
+		Scheme:   "https",
+		Host:     "console.aws.amazon.com",
+		Path:     path.Join("rds", "home"),
```

Comment:
This path is not going to have a leading `/`. It doesn't matter much right now as there is nothing that accesses the path except `URL.String()` and that puts in a slash, but it is different from what `url.Parse()` would return.

Since `path.Join()` is specified to use forward slashes to join elements, perhaps it's just easier to use the string literal `"/rds/home"`?